### PR TITLE
updates: implement inventoryGroupDevicesInfo

### PR DESCRIPTION
**Pull request by ldjebran**
# Description
In the context of inventory group details view we want to update devices of an inventory group. But before that we to need to retrieve validation data and the device uuids related to this group.

FIXES: https://issues.redhat.com/browse/THEEDGE-3539

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

File: pkg/models/updates_api.go [hunk 1]
```diff
@@ -85,3 +85,13 @@ type RecipientNotificationAPI struct {
 	IgnoreUserPreferences bool     `json:"ignore_user_preferences" example:"false"` // notification recipient to ignore user preferences
 	Users                 []string `json:"users" example:"user-id"`                 // notification recipient users
 } // @name RecipientNotification
+
+// InventoryGroupDevicesUpdateInfoResponseAPI is the inventory group update info
+type InventoryGroupDevicesUpdateInfoResponseAPI struct {
+	GroupUUID      string   `json:"group_uuid" example:"b579a578-1a6f-48d5-8a45-21f2a656a5d4"`                                                // the inventory group id
```
Comment (mgold1234) [inline pkg/models/updates_api.go:91]: this is real uuid?

File: pkg/routes/updates.go [hunk 2]
```diff
@@ -486,3 +538,46 @@ func ValidateGetUpdatesFilterParams(next http.Handler) http.Handler {
...
+// GetInventoryGroupDevicesUpdateInfo returns inventory group update info
+// @Summary      Gets the inventory group update info
+// @ID           GetInventoryGroupDevicesUpdateInfo
+// @Description  Gets the inventory group update info
+// @Tags         Updates (Systems)
+// @Accept       json
+// @Produce      json
+// @Param        GroupUUID  path  string    true  "a unique uuid to identify the inventory group"
+// @Success      200 {object} models.InventoryGroupDevicesUpdateInfoResponseAPI	"The requested inventory group update info"
+// @Failure      400 {object} errors.BadRequest	"The request sent couldn't be processed"
```
Comment (mgold1234) [inline pkg/routes/updates.go:551]: can you align all descriptions?

Comment (ldjebran) [reply pkg/routes/updates.go:551]: I see that they are all already aligned :smiley:

File: pkg/models/updates_api.go (hunk 1 above)
Comment (ldjebran) [reply pkg/models/updates_api.go:91]: yes

**Review by acosferreira — APPROVED** [review]
lgtm

**Review by acosferreira — DISMISSED** [review]

**Review by omaciel — APPROVED** [review]
/lgtm

**Review by acosferreira — APPROVED** [review]

**Pull request merged**